Dockerfile

```diff
@@ -106,12 +107,12 @@ RUN echo 'options("repos"="http://cloud.r-project.org")' >> /usr/local/lib/R/etc
 
 ## More verbose UBSAN/SAN output (cf #3) -- this is still somewhat speculative
 ## Entry copied from Prof Ripley's setup described at http://www.stats.ox.ac.uk/pub/bdr/memtests/README.txt
-ENV ASAN_OPTIONS 'alloc_dealloc_mismatch=0:detect_leaks=0:detect_odr_violation=0' 
+ENV ASAN_OPTIONS='alloc_dealloc_mismatch=0:detect_leaks=0:detect_odr_violation=0'
 
 RUN cd /usr/local/bin \
 	&& mv R Rdevel \
 	&& mv Rscript Rscriptdevel \
 	&& ln -s Rdevel RD \
 	&& ln -s Rscriptdevel RDscript
 
-
+CMD ["Rdevel"]
```

Comment:
No, I think I prefer shell.

Reply:
* Currently it defaults to the non-instrumented version of R...
* Which shell would you prefer?  `bash`?

Reply:
> Currently it defaults to the non-instrumented version of R...

Via the underlying container.  I have aliases for the various `docker` calls I make but muscle-memory makes me add ` bash` after each and ever container name ...

Reply:
Updated now to `CMD ["bash"]`